### PR TITLE
doc: add minutes for CTC meeting 9 Nov 2016

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

**Comment by Trott** [comment]
Quick scan LGTM. Thanks for doing this and also running the meeting today! :tada:

/cc @joshgav

**Review by Trott — APPROVED** [review]

**Comment by joshgav** [comment]
Yes thanks! I made arrangements so I won't miss the new time in the future ;)

Sent from Outlook Mobilehttps://aka.ms/blhgte

**Comment by mhdawson** [comment]
@nodejs/ctc, Plan to land this tomorrow unless there are objections/comments.

**Comment by williamkapke** [comment]
Can we change the name to use `YYYY-MM-DD.md`? (it's missing a `0` ATM)

File: meetings/2016-11-9.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+# Node Foundation CTC Meeting 2016-11-09
+
+## Links
+
+* **Audio Recording**: TBP
```
Comment (williamkapke) [inline meetings/2016-11-9.md]: Did this not make it to YouTube? I can't find it.

If not, and someone has the recording- send it my way and I'll upload it.

Comment (williamkapke) [reply meetings/2016-11-9.md]: Here it is: https://www.youtube.com/watch?v=xr8YKTnzu-4

<del>It is titled wrong on YouTube. I'll fix it now.</del> Fixed!

**Comment by mhdawson** [comment]
updated to address comments from @williamkapke

**Review by williamkapke — APPROVED** [review]

**Comment by mhdawson** [comment]
Landed as de89b5b86d75cbe7c859ac9097745cca331f453f

**Pull request closed without merging**